squad/core/migrations/0121_auto_20190711_1829.py

```diff
@@ -0,0 +1,25 @@
+# -*- coding: utf-8 -*-
```

Comment:
Could you rename the migration so the file name is descriptive

Reply:
definitely forgot that one, Thanks for heads up